addons/gdUnit3/plugin.gd

```diff
@@ -36,12 +41,9 @@ func _exit_tree():
 	_gd_inspector.free()
 	_gd_console.free()
 	_server_node.free()
+	if _update_tool and not _update_tool.is_update_in_progress():
```

Comment:
```suggestion
        # only delete and free the update tool when is not in use otherwise it will break the update execution
	if _update_tool and not _update_tool.is_update_in_progress():
```